app/Http/Controllers/Admin/AdminReportController.php

```diff
@@ -665,6 +669,24 @@ public function reportsApiAll(Request $request)
         return $reports;
     }
 
+    public function reportsApiRemote(Request $request)
+    {
+        $filter = $request->input('filter') == 'closed' ? 'closed' : 'open';
+
+        $reports = AdminRemoteReport::collection(
+            RemoteReport::orderBy('id', 'desc')
+                ->when($filter, function ($q, $filter) {
+                    return $filter == 'open' ?
+                    $q->whereNull('action_taken_at') :
+                    $q->whereNotNull('action_taken_at');
+                })
+                ->cursorPaginate(6)
```

Comment:
Showing only 6 at a time seems like an odd choice, considering some instances may be receiving hundreds of reports a day (this was the case for us at switter.at)